spec/unit/rest/client_spec.rb

```diff
@@ -50,4 +50,11 @@
       end
     end
   end
+
+  context 'request_id generation' do
+    let(:client_options) { {key: 'appid.keyuid:keysecret', add_request_ids: true} }
+    it 'includes request_id in URL' do
+      expect(subject.add_request_ids).to eql(true)
+    end
+  end
```

Comment:
If you're going to add a test, may as well be one that checks the result (i.e. uses webmock to check that a `request_id` actually gets added to the querystring for rest requests, and checks timeout exceptions to make sure that the request ID gets printed with those), rather than just checks that an instance variable got set correctly, no?